wordpress-mobile/WordPress-iOS
# Offline Mode: Remove Edit button from post success screen

- Remove "Edut" button from post publish success screen
- Rename "PostPostViewController`

To test:

- Publish a post
- Tap "View" in the success notice
- **Verify** that the success screen is shown
- **Verify** that all buttons on the success screen still work

## Regression Notes
1. Potential unintended areas of impact: Post Success Screen
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


Testing checklist:
